packages/material-ui/src/utils/chainPropType.test.js

```diff
@@ -1,37 +1,45 @@
 import { assert } from 'chai';
 import PropTypes from 'prop-types';
+import { mock } from 'sinon';
 import chainPropType from './chainPropType';
 
 describe('chainPropType', () => {
-  const props = {};
   const propName = 'children';
+  const props = { [propName]: 'some string' };
   const componentName = 'ComponentName';
   const location = 'prop';
-  const propFullName = null;
+  let warning;
+
+  beforeEach(() => {
+    warning = mock(console).expects('error');
```

Comment:
I remember now why we didn't do it this way. we can no longer use  console.log. Let's us our helper.